resources/icetes_schctrl.py

```diff
@@ -337,6 +337,20 @@ def on_after_new_environment_warmup_is_complete(self, state) -> int:
             state,
             num_tanks_hndl
         )
+        
+        # get storage medium
+        strg_med_hndl = self.api.exchange.get_variable_handle(
+            state,
+            "Schedule Value",
+            "Storage Medium"
+        )
+        
+        reverse_lookup = {5: "Water", 6: "SimpleWater", 7: "PCM2X2A"}
```

Comment:
Here are where the magic number gets converted into a string. If the number 8 is found in the schedule, there will be a problem.